multimodal-search-pdf/README.md

```diff
@@ -66,148 +43,20 @@ pip install -r requirements.txt
+| ``python app.py -t query_restful`` | To run a query Flow exposing a restful API (support search with image, text or PDFs) |
+| ``python app.py -t query_text`` | To run a query Flow exposing a grpc interface (support search with text) |
 
 ## Start the Server
 
 ``` bash
-python app.py -t query_restful
+python app.py -t index
+python app.py -t query_text
 ```
 
 ## Query via REST API
 
 When the REST gateway is enabled, Jina uses the [data URI scheme](https://en.wikipedia.org/wiki/Data_URI_scheme) to represent multimedia data. Simply organize your picture(s) into this scheme and send a POST request to `http://0.0.0.0:45670/api/search`, e.g.:
 
 ``` bash
-curl --request POST -d '{"top_k": 10, "mode": "search",  "data": ["jina hello multimodal"]}' -H 'Content-Type: application/json' 'http://0.0.0.0:45670/api/search'
-```
-
-<details>
-<summary> The results will be like: </summary> 
-
-```json
-{
-   "requestId":"b5813834-5f42-4f6c-a313-0f397ef5cf12",
-   "search":{
-      "docs":[
-         {
-            "id":"2cd45334-98dd-11eb-85a3-38f9d3eb0f3c",
-            "weight":1.0,
-            "matches":[
-               {
-                  "id":"f61571fc-9706-11eb-ba29-38f9d3eb0f3c",
-                  "uri":"toy_data/blog1.pdf",
-                  "mimeType":"application/pdf",
-                  "score":{
-                     "value":1.0,
-                     "opName":"SimpleAggregateRanker",
-                     "refId":"2cd45334-98dd-11eb-85a3-38f9d3eb0f3c"
-                  },
-                  "adjacency":1,
-                  "contentHash":"3b46f0781f207674"
-               },
-               {
-                  "id":"f618041c-9706-11eb-ba29-38f9d3eb0f3c",
-                  "uri":"toy_data/blog3.pdf",
-                  "mimeType":"application/pdf",
-                  "score":{
-                     "value":0.83538496,
-                     "opName":"SimpleAggregateRanker",
-                     "refId":"2cd45334-98dd-11eb-85a3-38f9d3eb0f3c"
-                  },
-                  "adjacency":1,
-                  "contentHash":"89a45525161ec448"
-               },
-               {
-                  "id":"f617ec70-9706-11eb-ba29-38f9d3eb0f3c",
-                  "uri":"toy_data/blog2.pdf",
-                  "mimeType":"application/pdf",
-                  "score":{
-                     "value":0.71500874,
-                     "opName":"SimpleAggregateRanker",
-                     "refId":"2cd45334-98dd-11eb-85a3-38f9d3eb0f3c"
-                  },
-                  "adjacency":1,
-                  "contentHash":"debde7ffbb210b46"
-               }
-            ],
-            "mimeType":"text/plain",
-            "text":"jina hello multimodal",
-            "contentHash":"f7beb6d4b839aa06"
-         }
-      ]
-   }
+curl --request POST -d '{"top_k": 10, "mode": "search",  "data": ["jina hello multimodal"]}' -H 'Content-Type: application/json' 'http://0.0.0.0:52018/api/search'
```

Comment:
The search URI here is wrong: http://0.0.0.0:52018/api/search
Both the path (it is not /api/search but /search)
And the port - please stick to one port throughout the example - I would suggest using 45678 as port

Reply:
https://github.com/jina-ai/examples/pull/629/commits/1bbcd41b3cc8c79841a261a5595fec0bb8806cad